### PR TITLE
style approved cards and location drop down

### PR DESCRIPTION
**Pull request by Peter-Springer**

### CODE REVIEW

**Comment by Peter-Springer** [comment]
Cards are visually different if a spike has been approved. The issue also suggested placing cards in different locations based on approval status however I feel this is covered by our filter functionality.
screen shot 2017-01-05 at 6 51 35 pm

**Comment by swanie21** [comment]
Is that blanched almond I see! 👀 You can definitely differentiate the approved/pending spikes now.

File: lib/components/AdminSpike.js [hunk 1]
```diff
@@ -4,7 +4,10 @@ import moment from 'moment';
 
 const AdminSpike = ({ spike, createSpike, updateSpike, deleteSpike }, key ) => {
   return (
-    <section className="SpikeCard AdminSpikeCard" key={key}>
+    <section
+      className={spike.appr? "ApprovedSpike" : "SpikeCard AdminSpikeCard"}
```
Comment (swanie21) [inline lib/components/AdminSpike.js]: Minor syntax correction with ternary - space before ?

File: lib/components/AdminSpike.js [hunk 2]
```diff
@@ -51,6 +53,12 @@ const AdminSpike = ({ spike, createSpike, updateSpike, deleteSpike }, key ) => {
             <option value='Vault Study'>Vault Study</option>
           </select>
         </section>
+        <p className="location-assignment">
+          Assigned Location:  <span>{spike.location}</span>
+        </p>
+        <p>Approval Status:
+          <span>{spike.appr ?  'Approved' : 'Pending'}</span>
```
Comment (swanie21) [inline lib/components/AdminSpike.js]: minor syntax correction - remove one space after ? and before 'Approved'

**Comment by swanie21** [comment]
Thanks for making those syntax changes Pete, good to merge

**Pull request merged**